src/main/resources/anniv5Resources/localization/eng/energyandechopack/Cardstrings.json

```
@@ -41,5 +37,10 @@
     "NAME": "Slowing",
     "DESCRIPTION": "At the start of your turn, gain [E] . NL Add !M! *Wounds into your hand.",
     "UPGRADE_DESCRIPTION": "At the start of your turn, gain [E] . NL Add a *Wound into your hand."
+  },
+  "${ModID}:Crystallize": {
+    "NAME": "Crystallize",
+    "DESCRIPTION": "Gain !M! Block for each [E] available.",
+    "EXTENDED_DESCRIPTION": ["Gain !M! Block for each [E] available. NL (Gain !B! Block.)"]
```

Small wording change here for base game consistency with cards like Stack.

```suggestion
    "EXTENDED_DESCRIPTION": ["Gain !M! Block for each [E] available. NL (Gains !B! Block.)"]
```

---

Though now that I've posted this, I see that Spirit Shield does it the way you have it here. Base game inconsistency... I like "Gains" better than "Gain" but you can argue for either one.